app/index.html

```diff
@@ -155,6 +155,59 @@ <h2 class="card-pf-title">
             </div>
 
           </div>
+          <!-- stack analyses-->
+          <div class="row row-cards-pf">
+            <div class="card-pf card-pf-accented">
+              <div class="card-pf-heading">
+                <h2 class="card-pf-title">
+                  Analysis of Stack
+                </h2>
+              </div>
+              <div class="card-pf-body">
+                <div id="stackAnalyses">
+                  <div class="blank-slate-pf">
+                    <h1>Select manifest file to upload:</h1>
+                    <input type="file" class="btn btn-primary" id="stackAnalysesFile" style="display:inline">
+                    <div class="col-lg-12 col-sm-12">
+                      <br />
+                      <button type="button" id="stacAnalysesFileUpload" class="btn btn-primary">Upload</button>
+                    </div>
+                  </div>
+                  <br />
+                  <div id="pomStatusSuccess">
+                     <a href="#" id="stackAnalysesAnchor" class="alert-link">View report</a>
```

Comment:
have made use of existing toast notification to show messages but to have action in toast was having scope issues so just had "view report" link in view.